UnityProject/Assets/Scripts/Items/ClothingItem.cs

```diff
@@ -78,82 +79,57 @@ public void SetColor(Color value)
 	public virtual void SetReference(GameObject item)
 	{
 		UpdateReferenceOffset();
-
 		if (item == null)
 		{
-			RemoveItemFromEquipment();
-		}
-		else
-		{
-			AddItemToEquipment(item);
-		}
-
-		UpdateReferenceOffset();
-	}
-
-	private void RemoveItemFromEquipment()
-	{
-		if (spriteHandler != null)
-		{
-			spriteHandler.Empty();
-		}
-
-		if (!InHands && GameObjectReference != null)
-		{
-			// did we take off clothing?
-			var unequippedClothing = GameObjectReference.GetComponent<ClothingV2>();
-
-			if (unequippedClothing == null)
+			if (spriteHandler != null)
```

Comment:
clothing needs to be put into a slot on the body part And handle from there but this can be later on